mobile/src/pages/Service.tsx

```diff
@@ -1,88 +1,176 @@
+import {SafeAreaView, StyleSheet, View, Dimensions} from 'react-native';
+import axios from 'axios';
+import {useAuth} from '../context/UserContext';
+import ServiceCard from '../components/ServiceList/serviceList';
+import HeaderBar from '../components/headerComponent';
 
-function Service(): React.JSX.Element {
+const Service = ({navigation}) => {
   const backgroundStyle = {
-    backgroundColor: Colors.darker,
+    backgroundColor: '#1C1B1F',
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
   };
 
+  const {token} = useAuth();
+
+  const Google = require('../../assets/Google.png');
+  const Discord = require('../../assets/Discord.png');
+  const Github = require('../../assets/Github.png');
+  const Spotify = require('../../assets/Spotify.png');
+
+  interface ServiceItem {
+    typeService: string;
+  }
+
+  const [listServices, setListServices] = useState<ServiceItem[]>();
+  const [stateGoogle, setStateGoogle] = useState('Not Connected');
+  const [stateDiscord, setStateDiscord] = useState('Not Connected');
+  const [stateGithub, setStateGithub] = useState('Not Connected');
+  const [stateSpotify, setStateSpotify] = useState('Not Connected');
+
+  useEffect(() => {
+    if (listServices === undefined) {
+      const handleCallServicesList = () => {
+        const storedToken = token;
+        axios
+          .get('http://10.0.2.2:8080/services', {
+            headers: {
+              token: storedToken,
+            },
+          })
+          .then(response => {
+            setListServices(response.data);
+          })
+          .catch(error => {
+            console.error(error);
+          });
+      };
+      handleCallServicesList();
+    } else {
+      let i = 0;
+      while (listServices[i]) {
+        if (listServices[i].typeService === 'GOOGLE') {
+          setStateGoogle('Connected');
+        } else if (listServices[i].typeService === 'DISCORD') {
+          setStateDiscord('Connected');
+        } else if (listServices[i].typeService === 'GITHUB') {
+          setStateGithub('Connected');
+        } else if (listServices[i].typeService === 'SPOTIFY') {
+          setStateSpotify('Connected');
+        }
+        i = i + 1;
+      }
+    }
+  }, [listServices, token]);
+
   return (
     <SafeAreaView style={backgroundStyle}>
-      <StatusBar
+      {/* <StatusBar
         barStyle={'light-content'}
         backgroundColor={backgroundStyle.backgroundColor}
+      /> */}
+      <HeaderBar
+        page="Services"
+        left_icon={[
+          {
+            image_url: require('../../assets/BackIcon.png'),
+            onPress: () => {
+              navigation.goBack();
+            },
+          },
+        ]}
       />
-      <View style={styles.listContainer}>
-        <ServiceList />
+      <View style={styles.servicesCardArea}>
+        <ServiceCard
+          serviceName={'Google'}
+          status={stateGoogle}
+          image={Google}
+          userName={''}
```

Comment:
Do you really need this broken feature ?

Reply:
Yes